.github/workflows/build.yml

```diff
@@ -8,8 +8,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: ljharb/actions/node/install@main
-      name: 'nvm install lts/* && npm install'
+    - uses: actions/setup-node@v4
       with:
         node-version: lts/*
-    - run: npm run build
+        cache: 'npm'
+    - run: npm ci
+    - run: node --run build
```

Comment:
we can't use node --run, because it won't run pre and post scripts.

Reply:
In most cases, it seems obvious to me that having each step clearly written down instead of relying on magic is the best idea.

Reply:
Yes, it's just that it's something voluntary. But I don't think we should base our decision on that. Because a task, a script, a step in the CI, a call to a script...